const static int zarrFormat

Any reason to have these variables declared as `const static`?
https://github.com/constantinpape/z5/blob/7868934c5513ea683299969de83b48137618cd2d/include/z5/metadata.hxx#L17-L20
It leads to this error:
```
ld: /tmp/cc5iNV25.o: in function `z5::DatasetMetadata::toJsonZarr(nlohmann::basic_json<std::map, std::vector, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, bool, long, unsigned long, double, std::allocator, nlohmann::adl_serializer>&) const':
main.cc:(.text._ZNK2z515DatasetMetadata10toJsonZarrERN8nlohmann10basic_jsonISt3mapSt6vectorNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEblmdSaNS1_14adl_serializerEEE[_ZNK2z515DatasetMetadata10toJsonZarrERN8nlohmann10basic_jsonISt3mapSt6vectorNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEblmdSaNS1_14adl_serializerEEE]+0x3cf): undefined reference to `z5::Metadata::zarrFormat'
ld: /tmp/cc5iNV25.o: in function `void z5::filesystem::writeMetadata<z5::filesystem::handle::File>(z5::handle::File<z5::filesystem::handle::File> const&, z5::Metadata const&)':
main.cc:(.text._ZN2z510filesystem13writeMetadataINS0_6handle4FileEEEvRKNS_6handle4FileIT_EERKNS_8MetadataE[_ZN2z510filesystem13writeMetadataINS0_6handle4FileEEEvRKNS_6handle4FileIT_EERKNS_8MetadataE]+0xc5): undefined reference to `z5::Metadata::zarrFormat'
collect2: error: ld returned 1 exit status
```

--- a/include/z5/metadata.hxx
+++ b/include/z5/metadata.hxx
@@ -14,12 +14,17 @@ namespace z5 {
     // general format
     struct Metadata {
         bool isZarr; // flag to specify whether we have a zarr or n5 array
-        const static int zarrFormat = 2;
-        const static int n5Major = 2;
-        const static int n5Minor = 0;
-        const static int n5Patch = 0;
+        int zarrFormat;
+        int n5Major;
+        int n5Minor;
+        int n5Patch;
 
-        Metadata(const bool isZarr) : isZarr(isZarr) {}
+        Metadata(const bool isZarr) : isZarr(isZarr),
+                                      zarrFormat(2),
+                                      n5Major(2),
+                                      n5Minor(0),
+                                      n5Patch(0)
+        {}
         inline std::string n5Format() const {
             return std::to_string(n5Major) + "." + std::to_string(n5Minor) + "." + std::to_string(n5Patch);
         }
@@ -49,7 +54,7 @@ namespace z5 {
         }
 
 
-        // empty constructur
+        // empty constructor
         DatasetMetadata() : Metadata(true)
         {}
 

